spec/sanity/blueprints.js

```diff
@@ -51,17 +51,17 @@ describe('blueprints page', function(){
         browser.sleep(1000).then(function(){ done(); });
     });
 
-    //it('should open node details panel when clicking on node in topology section', function(done) {
-    //    logger.trace('start blueprint topology section test');
-    //    components.ui.blueprints.IndexPage.goToBlueprint({'name' : DEPLOY_BLUEPRINT_NAME});
-    //    components.ui.blueprints.BlueprintPage.goToSection('Topology');
-    //    components.ui.blueprints.BlueprintPage.Topology.clickNode('mongod_host');
-    //    components.ui.blueprints.BlueprintPage.FloatingPanel.goToSection('Properties');
-    //    components.ui.blueprints.BlueprintPage.FloatingPanel.goToSection('Relationships');
-    //    components.ui.blueprints.BlueprintPage.FloatingPanel.close();
-    //
-    //    browser.sleep(1000).then(function(){ done(); });
-    //});
+    it('should open node details panel when clicking on node in topology section', function(done) {
+        logger.trace('start blueprint topology section test');
+        components.ui.blueprints.IndexPage.goToBlueprint({'name' : testConf.blueprints.blueprintToRead});
+        components.ui.blueprints.BlueprintPage.goToSection('Topology');
+        components.ui.blueprints.BlueprintPage.Topology.clickNode('mongod');
+        components.ui.blueprints.BlueprintPage.FloatingPanel.goToSection('Properties');
```

Comment:
I think we should refactor the code a bit, to make it more maintainable and replace goToSection( 'name' ) to "goToProperties" and "goToRelationships" and so on.. 

==> no hard coded values
==> if we ever test translation, the implementation will change and it will be hard to change it across the board written like this. 

Plus, I would change the name from "FloatingPanel" to "PropertiesPanel" - more functional. 
If tomorrow the panel's design changes and it is not floating anymore, the name would not make sense.